curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b8.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 66c8ba975ee7230e29f6c4b8
+title: Step 17
+challengeType: 1
+dashedName: step-17
+---
+
+# --description--
+
+Use `const` to create a `finalMessage` variable and set it to \` `Your credit score is ${creditScore} and your annual income is ${annualIncome}. So, ${qualificationMessage}` \`.
```

Comment:
same thing here.
campers should be allowed to input any correct answer.
we should only check that they got the correct string format.
how, they choose to concatenate strings, whether it be template literals or `+` operator is up to them